### ISSUE TITLE
Deprecated escape sequence

### ISSUE THREAD

**Issue by lordmauve** [requests/requests-kerberos#132]
Importing `requests-kerberos` under Python 3.6+, with deprecation warnings enabled, prints

```
requestskerberos/0/12/0/dist/lib/python3.6/requests_kerberos/kerberos_.py:85: DeprecationWarning: invalid escape sequence \s
  regex = re.compile('(?:.*,)*\s*Negotiate\s*([^,]*),?', re.I)
```

**Comment by frozencemetery** [comment]
This would be fixed by #121 if that merged.

**Comment by lordmauve** [comment]
Fixed by #121.
